actions/patch_node_handler.go

```diff
@@ -30,6 +31,21 @@ func (h *patchNodeHandler) Handle(ctx context.Context, data interface{}) error {
 	if !ok {
 		return fmt.Errorf("unexpected type %T for delete patch handler", data)
 	}
+	for k := range req.Labels {
+		if k == "" {
+			return errors.New("labels contain entry with empty key")
```

Comment:
do we have similar validation before scheduling the action?

Reply:
Will add validation, thanks for suggestion.